lib/generateMarkdown.js

```diff
@@ -405,15 +404,17 @@ function getEnumString(schema, type) {
  * Gets the string describing the possible enum values, if they are defined within a JSON anyOf object.
  * @param  {object} schema The schema object that may be of an enum type.
  * @param  {string} type The name of the object type for the enum values (e.g. string, integer, etc..)
+ * @param  {integer} depth How deep the bullet points for enum values should be.  Maximum is 2.
  * @return {string} A string that enumerates all the possible enum values for this schema object.
  */
-function getAnyOfEnumString(schema, type) {
+function getAnyOfEnumString(schema, type, depth) {
     var propertyAnyOf = schema['anyOf'];
     if (!defined(propertyAnyOf)) {
         return undefined;
     }
 
     var allowedValues = '';
+    var prefix = '\n' + ('      '.substring(0, depth * 3)) + '* ';
```

Comment:
Might just be me, but I think this:
`('      '.substring(0, depth * 3))`
would probably be a little easier to read as:
`' '.repeat(depth * 3)`